packages/common-test-utils/src/noteUtils.ts

```diff
@@ -89,6 +90,16 @@ export class TestNoteFactory {
     }
     return noteProps;
   }
+
+  toNotePropsDict(notes: NoteProps[]): NotePropsDict {
+    const dict: NotePropsDict = {};
+
+    for (const note of notes) {
```

Comment:
nitpick -> for iteration, generally prefer `notes.forEach( note => {})` syntax as its more terse and common with es6